expressions_test.go

```diff
@@ -55,5 +55,21 @@ func (s *Zuite) TestSelectors() {
 		slice, ok := actual.(*Slice)
 		require.True(s.T(), ok)
 		require.Equal(s.T(), []Value{alice}, slice.Elements())
+		require.Equal(s.T(), &SliceType{alice.Type()}, slice.Type())
```

Comment:
Here, `alice.Type()` is `&TextType{}`, correct? Would rather have this explicitly stated.